tracing-env-logger/examples/hyper-echo.rs

```diff
@@ -124,15 +122,16 @@ fn main() {
 
     tracing::subscriber::with_default(subscriber, || {
         let addr: ::std::net::SocketAddr = ([127, 0, 0, 1], 3000).into();
-        let server_span = span!(Level::TRACE, "server", local = &field::debug(addr));
+        let server_span = span!(Level::TRACE, "server", local = %addr);
+        let _enter = server_span.enter();
         let server = tokio::net::TcpListener::bind(&addr)
             .expect("bind")
             .incoming()
             .fold(Http::new(), move |http, sock| {
                 let span = span!(
                     Level::TRACE,
                     "connection",
-                    remote = &field::debug(&sock.peer_addr().unwrap())
+                    remote = %sock.peer_addr().unwrap()
```

Comment:
Shouldn't this be `?` if it is to stay the same?

Reply:
whoops, will fix that